storage-resize-images/extension.yaml

```diff
@@ -191,14 +193,14 @@ params:
   - param: EXCLUDE_PATH_LIST
     label: List of absolute paths not included for resized images
     description: >
-      Ensure storage-resize-images does *not* resize images in _specific locations_ in your Storage bucket by 
-      supplying a comma-separated list of absolute paths. For example, to *exclude* the images 
+      Ensure storage-resize-images does *not* resize images in _specific locations_ in your Storage bucket by
+      supplying a comma-separated list of absolute paths. For example, to *exclude* the images
       stored in the `/users/pictures` and `/restaurants/menuItems` directories, specify the paths `/users/pictures,/restaurants/menuItems`.
-      
-      You may also use wildcard notation for directories in the path. For example, `/users/*/pictures` 
-      would exclude `/users/profile/pictures/image.png` as well as `/users/profile/pictures/any/sub/directory/image.png`. 
-      
-      If you prefer to resize every image uploaded to your Storage bucket, 
+
+      You may also use wildcard notation for directories in the path. For example, `/users/*/pictures`
+      would exclude `/users/profile/pictures/image.png` as well as `/users/profile/pictures/any/sub/directory/image.png`.
+
+      If you prefer to resize every image uploaded to your Storage bucket,
```

Comment:
Oops, seems like my IDE got rid of some of the extra whitespaces in those yaml files.

I can revert these changes if needed.